src/common/shutdown_state.c

```diff
@@ -42,6 +42,7 @@
 #include "out.h"
 #include "util.h"
 #include "os_deep.h"
+#include "set.h"
```

Comment:
I think this include is unnecessary - "set.h" is already included in line above ( "os_deep.h")